x-pack/legacy/plugins/apm/public/context/UrlParamsContext/resolveUrlParams.ts

```diff
@@ -79,6 +91,7 @@ export function resolveUrlParams(location: Location, state: TimeUrlParams) {
     errorGroupId,
 
     // ui filters
-    environment
+    environment,
+    ...localUiFilters
```

Comment:
Why treat local filters differently from other url params? Would be easier to read if they are written out explicitly.

Reply:
same as above - trying to avoid writing out those variable names in various places. I'm having a hard time seeing how writing them out in 3-4 places helps our codebase. Do you think it'd be better?